Configuration/StandardSequences/python/ValidationCosmics_cff.py

```diff
@@ -10,8 +10,6 @@
 
 from Validation.Configuration.globalValidationCosmics_cff import *
 
-prevalidation = cms.Sequence(globalPrevalidationCosmics)
+prevalidationCosmics = cms.Sequence(globalPrevalidationCosmics * cms.SequencePlaceholder("mix"))
```

Comment:
Hi, as agreed I have reverted back to the default cosmics validation configuration.